src/EnergyPlus/WindowManager.cc

```diff
@@ -5833,7 +5833,7 @@ namespace Window {
         DPhi = 5.0 * Constant::DegToRadians;
 
         // Integrate from -90 to 0 deg
-        for (int IPhi = 1; IPhi <= 18; ++IPhi) {
+        for (int IPhi = 1; IPhi <= (Material::MaxProfAngs / 2); ++IPhi) {
```

Comment:
Noticed that these are literals so changed to use the `constexpr int MaxProfAngs`. That is all.